pyface/tasks/task_window.py

```diff
@@ -348,6 +347,9 @@ def _destroy_state(self, state):
             if state.menu_bar_manager:
                 state.menu_bar_manager.destroy()
             for tool_bar_manager in state.tool_bar_managers:
+                # FIXME: toolbars were not getting removed, but this isn't the right place for it
+                pane = self._window_backend.window._aui_manager.GetPane(tool_bar_manager.id)
```

Comment:
@mdickinson pointed out that this seems to be assuming a Wx backend and will likely break under Qt.